packages/server/graphql/private/mutations/processRecurrence.ts

```diff
@@ -33,16 +33,14 @@ const startRecurringTeamPrompt = async (
     day: 'numeric'
   })
 
+  const rrule = RRule.fromString(meetingSeries.recurrenceRule)
+  const nextMeetingStartDate = rrule.after(startTime)
   const meeting = await safeCreateTeamPrompt(teamId, facilitatorId, r, dataLoader, {
     name: `${meetingSeries.title} - ${formattedDate}`,
-    scheduledEndTime: new Date(startTime.getTime() + ms(`${meetingSeries.duration}m`)),
+    scheduledEndTime: nextMeetingStartDate,
```

Comment:
I changed the way we calculate the next meeting start date because now, that we support multiple recurrence configurations, we can't rely on fixed meeting duration. Imagine the following scenario:

The meeting is set to recur weekly on Fridays and Saturdays. It's ideal to assume that the meeting will last only 24h, and I'd expect the meeting to last the whole period before the next one starts. this means Saturday's meeting will last almost a week and will end just before Friday's meeting starts. Friday's meeting will last only 24h and will end on Saturday (just before Saturday's meeting starts)

@jmtaber129 @enriquesanchez wdyt?

Reply:
>I'd expect the meeting to last the whole period before the next one starts

This sounds right to me 👍